libraries/Microsoft.Bot.Connector/Authentication/CertificateAppCredentials.cs

```diff
@@ -104,7 +104,7 @@ protected override Lazy<AdalAuthenticator> BuildAuthenticator()
                 new AdalAuthenticator(
                     this.clientCertificate,
                     this.sendX5c,
-                    new OAuthConfiguration() { Authority = OAuthEndpoint, Scope = OAuthScope },
+                    new OAuthConfiguration() { Authority = OAuthEndpoint, ValidateAuthority = ValidateAuthority, Scope = OAuthScope },
```

Comment:
Is this backwards compatible?

Reply:
It was supposed to be, do you see a problem?